From e7dfb803ed5fd7d586a5c14859487a5420d19614 Mon Sep 17 00:00:00 2001
From: "Huang, Tao" <huangtao@rock-chips.com>
Date: Mon, 20 Feb 2017 15:49:03 +0800
Subject: [PATCH] arm64: dts: rk3399: rename android to android-6.0

Except dts of VR.
The md5sum is identical after rename, so this commit is safe.

Change-Id: I9ec324355ae67bbe2bb626090402ae797de13d92
Signed-off-by: Huang, Tao <huangtao@rock-chips.com>
---
 arch/arm64/boot/dts/rockchip/Makefile         | 20 +++++++++----------
 ...9-android.dtsi => rk3399-android-6.0.dtsi} |  0
 ...oid.dts => rk3399-box-808-android-6.0.dts} |  2 +-
 ...9-box.dtsi => rk3399-box-android-6.0.dtsi} |  2 +-
 ...v1.dts => rk3399-box-rev1-android-6.0.dts} |  2 +-
 .../dts/rockchip/rk3399-box-rev1-disvr.dts    |  2 +-
 ...v2.dts => rk3399-box-rev2-android-6.0.dts} |  2 +-
 .../dts/rockchip/rk3399-box-rev2-disvr.dts    |  2 +-
 .../dts/rockchip/rk3399-disvr-android.dts     |  2 +-
 ...id.dts => rk3399-evb-rev1-android-6.0.dts} |  2 +-
 ...id.dts => rk3399-evb-rev2-android-6.0.dts} |  2 +-
 ...ts => rk3399-evb-rev3-android-6.0-edp.dts} |  2 +-
 ...id.dts => rk3399-evb-rev3-android-6.0.dts} |  2 +-
 ...oid.dts => rk3399-mid-818-android-6.0.dts} |  2 +-
 .../boot/dts/rockchip/rk3399-rv1-android.dts  |  2 +-
 ...99-sapphire-excavator-box-android-6.0.dts} |  2 +-
 .../rk3399-sapphire-excavator-disvr.dts       |  2 +-
 ...99-sapphire-excavator-edp-android-6.0.dts} |  2 +-
 .../boot/dts/rockchip/rk3399-vr-android.dts   |  2 +-
 19 files changed, 27 insertions(+), 27 deletions(-)
 rename arch/arm64/boot/dts/rockchip/{rk3399-android.dtsi => rk3399-android-6.0.dtsi} (100%)
 rename arch/arm64/boot/dts/rockchip/{rk3399-box-808-android.dts => rk3399-box-808-android-6.0.dts} (99%)
 rename arch/arm64/boot/dts/rockchip/{rk3399-box.dtsi => rk3399-box-android-6.0.dtsi} (99%)
 rename arch/arm64/boot/dts/rockchip/{rk3399-box-rev1.dts => rk3399-box-rev1-android-6.0.dts} (98%)
 rename arch/arm64/boot/dts/rockchip/{rk3399-box-rev2.dts => rk3399-box-rev2-android-6.0.dts} (98%)
 rename arch/arm64/boot/dts/rockchip/{rk3399-evb-rev1-android.dts => rk3399-evb-rev1-android-6.0.dts} (98%)
 rename arch/arm64/boot/dts/rockchip/{rk3399-evb-rev2-android.dts => rk3399-evb-rev2-android-6.0.dts} (99%)
 rename arch/arm64/boot/dts/rockchip/{rk3399-evb-rev3-android-edp.dts => rk3399-evb-rev3-android-6.0-edp.dts} (98%)
 rename arch/arm64/boot/dts/rockchip/{rk3399-evb-rev3-android.dts => rk3399-evb-rev3-android-6.0.dts} (99%)
 rename arch/arm64/boot/dts/rockchip/{rk3399-mid-818-android.dts => rk3399-mid-818-android-6.0.dts} (99%)
 rename arch/arm64/boot/dts/rockchip/{rk3399-sapphire-excavator-box.dts => rk3399-sapphire-excavator-box-android-6.0.dts} (99%)
 rename arch/arm64/boot/dts/rockchip/{rk3399-sapphire-excavator-edp.dts => rk3399-sapphire-excavator-edp-android-6.0.dts} (99%)

diff --git a/arch/arm64/boot/dts/rockchip/Makefile b/arch/arm64/boot/dts/rockchip/Makefile
index f1044591ec73..8c69677edf09 100644
--- a/arch/arm64/boot/dts/rockchip/Makefile
+++ b/arch/arm64/boot/dts/rockchip/Makefile
@@ -3,34 +3,34 @@ dtb-$(CONFIG_ARCH_ROCKCHIP) += rk3366-fpga.dtb
 dtb-$(CONFIG_ARCH_ROCKCHIP) += rk3366-tb.dtb
 dtb-$(CONFIG_ARCH_ROCKCHIP) += rk3368-r88.dtb
 dtb-$(CONFIG_ARCH_ROCKCHIP) += rk3368-tb-sheep.dtb
-dtb-$(CONFIG_ARCH_ROCKCHIP) += rk3399-box-808-android.dtb
-dtb-$(CONFIG_ARCH_ROCKCHIP) += rk3399-box-rev1.dtb
+dtb-$(CONFIG_ARCH_ROCKCHIP) += rk3399-box-808-android-6.0.dtb
+dtb-$(CONFIG_ARCH_ROCKCHIP) += rk3399-box-rev1-android-6.0.dtb
 dtb-$(CONFIG_ARCH_ROCKCHIP) += rk3399-box-rev1-disvr.dtb
-dtb-$(CONFIG_ARCH_ROCKCHIP) += rk3399-box-rev2.dtb
+dtb-$(CONFIG_ARCH_ROCKCHIP) += rk3399-box-rev2-android-6.0.dtb
 dtb-$(CONFIG_ARCH_ROCKCHIP) += rk3399-box-rev2-disvr.dtb
 dtb-$(CONFIG_ARCH_ROCKCHIP) += rk3399-box-rev1-next.dtb
 dtb-$(CONFIG_ARCH_ROCKCHIP) += rk3399-box-rev2-next.dtb
 dtb-$(CONFIG_ARCH_ROCKCHIP) += rk3399-disvr-android.dtb
-dtb-$(CONFIG_ARCH_ROCKCHIP) += rk3399-evb-rev1-android.dtb
+dtb-$(CONFIG_ARCH_ROCKCHIP) += rk3399-evb-rev1-android-6.0.dtb
 dtb-$(CONFIG_ARCH_ROCKCHIP) += rk3399-evb-rev1-android-next.dtb
 dtb-$(CONFIG_ARCH_ROCKCHIP) += rk3399-evb-rev1-cros.dtb
-dtb-$(CONFIG_ARCH_ROCKCHIP) += rk3399-evb-rev2-android.dtb
+dtb-$(CONFIG_ARCH_ROCKCHIP) += rk3399-evb-rev2-android-6.0.dtb
 dtb-$(CONFIG_ARCH_ROCKCHIP) += rk3399-evb-rev2-android-next.dtb
 dtb-$(CONFIG_ARCH_ROCKCHIP) += rk3399-evb-rev2-cros.dtb
-dtb-$(CONFIG_ARCH_ROCKCHIP) += rk3399-evb-rev3-android.dtb
-dtb-$(CONFIG_ARCH_ROCKCHIP) += rk3399-evb-rev3-android-edp.dtb
+dtb-$(CONFIG_ARCH_ROCKCHIP) += rk3399-evb-rev3-android-6.0.dtb
+dtb-$(CONFIG_ARCH_ROCKCHIP) += rk3399-evb-rev3-android-6.0-edp.dtb
 dtb-$(CONFIG_ARCH_ROCKCHIP) += rk3399-evb-rev3-android-edp-next.dtb
 dtb-$(CONFIG_ARCH_ROCKCHIP) += rk3399-evb-rev3-cros.dtb
 dtb-$(CONFIG_ARCH_ROCKCHIP) += rk3399-fpga.dtb
 dtb-$(CONFIG_ARCH_ROCKCHIP) += rk3399-gru-gru.dtb
 dtb-$(CONFIG_ARCH_ROCKCHIP) += rk3399-gru-kevin-r0.dtb
 dtb-$(CONFIG_ARCH_ROCKCHIP) += rk3399-gru-kevin-r1.dtb
-dtb-$(CONFIG_ARCH_ROCKCHIP) += rk3399-mid-818-android.dtb
+dtb-$(CONFIG_ARCH_ROCKCHIP) += rk3399-mid-818-android-6.0.dtb
 dtb-$(CONFIG_ARCH_ROCKCHIP) += rk3399-mid-818-android-next.dtb
 dtb-$(CONFIG_ARCH_ROCKCHIP) += rk3399-rv1-android.dtb
-dtb-$(CONFIG_ARCH_ROCKCHIP) += rk3399-sapphire-excavator-box.dtb
+dtb-$(CONFIG_ARCH_ROCKCHIP) += rk3399-sapphire-excavator-box-android-6.0.dtb
 dtb-$(CONFIG_ARCH_ROCKCHIP) += rk3399-sapphire-excavator-disvr.dtb
-dtb-$(CONFIG_ARCH_ROCKCHIP) += rk3399-sapphire-excavator-edp.dtb
+dtb-$(CONFIG_ARCH_ROCKCHIP) += rk3399-sapphire-excavator-edp-android-6.0.dtb
 dtb-$(CONFIG_ARCH_ROCKCHIP) += rk3399-sapphire-excavator-edp-next.dtb
 dtb-$(CONFIG_ARCH_ROCKCHIP) += rk3399-sapphire-excavator-linux.dtb
 dtb-$(CONFIG_ARCH_ROCKCHIP) += rk3399-tve1205g.dtb
diff --git a/arch/arm64/boot/dts/rockchip/rk3399-android.dtsi b/arch/arm64/boot/dts/rockchip/rk3399-android-6.0.dtsi
similarity index 100%
rename from arch/arm64/boot/dts/rockchip/rk3399-android.dtsi
rename to arch/arm64/boot/dts/rockchip/rk3399-android-6.0.dtsi
diff --git a/arch/arm64/boot/dts/rockchip/rk3399-box-808-android.dts b/arch/arm64/boot/dts/rockchip/rk3399-box-808-android-6.0.dts
similarity index 99%
rename from arch/arm64/boot/dts/rockchip/rk3399-box-808-android.dts
rename to arch/arm64/boot/dts/rockchip/rk3399-box-808-android-6.0.dts
index dc5644e85a6f..47cdb21c4d39 100644
--- a/arch/arm64/boot/dts/rockchip/rk3399-box-808-android.dts
+++ b/arch/arm64/boot/dts/rockchip/rk3399-box-808-android-6.0.dts
@@ -45,7 +45,7 @@
 #include <dt-bindings/pwm/pwm.h>
 #include <dt-bindings/input/input.h>
 #include "rk3399.dtsi"
-#include "rk3399-android.dtsi"
+#include "rk3399-android-6.0.dtsi"
 
 / {
 	model = "Rockchip RK3399 Box Board v1 (Android)";
diff --git a/arch/arm64/boot/dts/rockchip/rk3399-box.dtsi b/arch/arm64/boot/dts/rockchip/rk3399-box-android-6.0.dtsi
similarity index 99%
rename from arch/arm64/boot/dts/rockchip/rk3399-box.dtsi
rename to arch/arm64/boot/dts/rockchip/rk3399-box-android-6.0.dtsi
index d14e77f8e621..caabb097cacb 100644
--- a/arch/arm64/boot/dts/rockchip/rk3399-box.dtsi
+++ b/arch/arm64/boot/dts/rockchip/rk3399-box-android-6.0.dtsi
@@ -42,7 +42,7 @@
 #include <dt-bindings/pwm/pwm.h>
 #include <dt-bindings/input/input.h>
 #include "rk3399.dtsi"
-#include "rk3399-android.dtsi"
+#include "rk3399-android-6.0.dtsi"
 #include "rk3399-opp.dtsi"
 
 / {
diff --git a/arch/arm64/boot/dts/rockchip/rk3399-box-rev1.dts b/arch/arm64/boot/dts/rockchip/rk3399-box-rev1-android-6.0.dts
similarity index 98%
rename from arch/arm64/boot/dts/rockchip/rk3399-box-rev1.dts
rename to arch/arm64/boot/dts/rockchip/rk3399-box-rev1-android-6.0.dts
index 60b2e9fe5c20..89f53d059eb1 100644
--- a/arch/arm64/boot/dts/rockchip/rk3399-box-rev1.dts
+++ b/arch/arm64/boot/dts/rockchip/rk3399-box-rev1-android-6.0.dts
@@ -41,7 +41,7 @@
  */
 
 /dts-v1/;
-#include "rk3399-box.dtsi"
+#include "rk3399-box-android-6.0.dtsi"
 
 / {
 	model = "Rockchip RK3399 Board rev1 (BOX)";
diff --git a/arch/arm64/boot/dts/rockchip/rk3399-box-rev1-disvr.dts b/arch/arm64/boot/dts/rockchip/rk3399-box-rev1-disvr.dts
index 02fb90e87817..e98d091827e0 100644
--- a/arch/arm64/boot/dts/rockchip/rk3399-box-rev1-disvr.dts
+++ b/arch/arm64/boot/dts/rockchip/rk3399-box-rev1-disvr.dts
@@ -41,7 +41,7 @@
  */
 
 /dts-v1/;
-#include "rk3399-box.dtsi"
+#include "rk3399-box-android-6.0.dtsi"
 
 / {
 	model = "Rockchip RK3399 Board rev1 (BOX)";
diff --git a/arch/arm64/boot/dts/rockchip/rk3399-box-rev2.dts b/arch/arm64/boot/dts/rockchip/rk3399-box-rev2-android-6.0.dts
similarity index 98%
rename from arch/arm64/boot/dts/rockchip/rk3399-box-rev2.dts
rename to arch/arm64/boot/dts/rockchip/rk3399-box-rev2-android-6.0.dts
index b8e0f821183d..b57536d949a3 100644
--- a/arch/arm64/boot/dts/rockchip/rk3399-box-rev2.dts
+++ b/arch/arm64/boot/dts/rockchip/rk3399-box-rev2-android-6.0.dts
@@ -41,7 +41,7 @@
  */
 
 /dts-v1/;
-#include "rk3399-box.dtsi"
+#include "rk3399-box-android-6.0.dtsi"
 
 / {
 	model = "Rockchip RK3399 Board rev2 (BOX)";
diff --git a/arch/arm64/boot/dts/rockchip/rk3399-box-rev2-disvr.dts b/arch/arm64/boot/dts/rockchip/rk3399-box-rev2-disvr.dts
index 2148de7bf04b..523ad9e01043 100644
--- a/arch/arm64/boot/dts/rockchip/rk3399-box-rev2-disvr.dts
+++ b/arch/arm64/boot/dts/rockchip/rk3399-box-rev2-disvr.dts
@@ -41,7 +41,7 @@
  */
 
 /dts-v1/;
-#include "rk3399-box.dtsi"
+#include "rk3399-box-android-6.0.dtsi"
 
 / {
 	model = "Rockchip RK3399 Board rev2 (BOX)";
diff --git a/arch/arm64/boot/dts/rockchip/rk3399-disvr-android.dts b/arch/arm64/boot/dts/rockchip/rk3399-disvr-android.dts
index 92664659e237..9e406cbd6202 100644
--- a/arch/arm64/boot/dts/rockchip/rk3399-disvr-android.dts
+++ b/arch/arm64/boot/dts/rockchip/rk3399-disvr-android.dts
@@ -44,7 +44,7 @@
 #include <dt-bindings/pwm/pwm.h>
 #include <dt-bindings/sensor-dev.h>
 #include "rk3399.dtsi"
-#include "rk3399-android.dtsi"
+#include "rk3399-android-6.0.dtsi"
 #include "rk3399-opp.dtsi"
 
 / {
diff --git a/arch/arm64/boot/dts/rockchip/rk3399-evb-rev1-android.dts b/arch/arm64/boot/dts/rockchip/rk3399-evb-rev1-android-6.0.dts
similarity index 98%
rename from arch/arm64/boot/dts/rockchip/rk3399-evb-rev1-android.dts
rename to arch/arm64/boot/dts/rockchip/rk3399-evb-rev1-android-6.0.dts
index 6c6b2ad3eb53..9152bf1a4036 100644
--- a/arch/arm64/boot/dts/rockchip/rk3399-evb-rev1-android.dts
+++ b/arch/arm64/boot/dts/rockchip/rk3399-evb-rev1-android-6.0.dts
@@ -43,7 +43,7 @@
 /dts-v1/;
 
 #include "rk3399-evb-rev1.dtsi"
-#include "rk3399-android.dtsi"
+#include "rk3399-android-6.0.dtsi"
 
 / {
 	model = "Rockchip RK3399 Evaluation Board v1 (Android)";
diff --git a/arch/arm64/boot/dts/rockchip/rk3399-evb-rev2-android.dts b/arch/arm64/boot/dts/rockchip/rk3399-evb-rev2-android-6.0.dts
similarity index 99%
rename from arch/arm64/boot/dts/rockchip/rk3399-evb-rev2-android.dts
rename to arch/arm64/boot/dts/rockchip/rk3399-evb-rev2-android-6.0.dts
index 2ab480c37f97..8b2106b7907b 100644
--- a/arch/arm64/boot/dts/rockchip/rk3399-evb-rev2-android.dts
+++ b/arch/arm64/boot/dts/rockchip/rk3399-evb-rev2-android-6.0.dts
@@ -43,7 +43,7 @@
 /dts-v1/;
 
 #include "rk3399-evb-rev2.dtsi"
-#include "rk3399-android.dtsi"
+#include "rk3399-android-6.0.dtsi"
 
 / {
 	model = "Rockchip RK3399 Evaluation Board v2 (Android)";
diff --git a/arch/arm64/boot/dts/rockchip/rk3399-evb-rev3-android-edp.dts b/arch/arm64/boot/dts/rockchip/rk3399-evb-rev3-android-6.0-edp.dts
similarity index 98%
rename from arch/arm64/boot/dts/rockchip/rk3399-evb-rev3-android-edp.dts
rename to arch/arm64/boot/dts/rockchip/rk3399-evb-rev3-android-6.0-edp.dts
index a8e053ace757..c234fc7afc50 100644
--- a/arch/arm64/boot/dts/rockchip/rk3399-evb-rev3-android-edp.dts
+++ b/arch/arm64/boot/dts/rockchip/rk3399-evb-rev3-android-6.0-edp.dts
@@ -43,7 +43,7 @@
 /dts-v1/;
 
 #include "rk3399-evb-rev3.dtsi"
-#include "rk3399-android.dtsi"
+#include "rk3399-android-6.0.dtsi"
 
 / {
 	model = "Rockchip RK3399 Evaluation Board v3 edp (Android)";
diff --git a/arch/arm64/boot/dts/rockchip/rk3399-evb-rev3-android.dts b/arch/arm64/boot/dts/rockchip/rk3399-evb-rev3-android-6.0.dts
similarity index 99%
rename from arch/arm64/boot/dts/rockchip/rk3399-evb-rev3-android.dts
rename to arch/arm64/boot/dts/rockchip/rk3399-evb-rev3-android-6.0.dts
index c0e513819c59..42050cb40ccb 100644
--- a/arch/arm64/boot/dts/rockchip/rk3399-evb-rev3-android.dts
+++ b/arch/arm64/boot/dts/rockchip/rk3399-evb-rev3-android-6.0.dts
@@ -43,7 +43,7 @@
 /dts-v1/;
 
 #include "rk3399-evb-rev3.dtsi"
-#include "rk3399-android.dtsi"
+#include "rk3399-android-6.0.dtsi"
 
 / {
 	model = "Rockchip RK3399 Evaluation Board v3 (Android)";
diff --git a/arch/arm64/boot/dts/rockchip/rk3399-mid-818-android.dts b/arch/arm64/boot/dts/rockchip/rk3399-mid-818-android-6.0.dts
similarity index 99%
rename from arch/arm64/boot/dts/rockchip/rk3399-mid-818-android.dts
rename to arch/arm64/boot/dts/rockchip/rk3399-mid-818-android-6.0.dts
index cbf8286b48d9..9c234e5af3bf 100644
--- a/arch/arm64/boot/dts/rockchip/rk3399-mid-818-android.dts
+++ b/arch/arm64/boot/dts/rockchip/rk3399-mid-818-android-6.0.dts
@@ -43,7 +43,7 @@
 /dts-v1/;
 
 #include "rk3399.dtsi"
-#include "rk3399-android.dtsi"
+#include "rk3399-android-6.0.dtsi"
 #include "rk3399-opp.dtsi"
 #include <dt-bindings/sensor-dev.h>
 #include <dt-bindings/pwm/pwm.h>
diff --git a/arch/arm64/boot/dts/rockchip/rk3399-rv1-android.dts b/arch/arm64/boot/dts/rockchip/rk3399-rv1-android.dts
index cc72207d2cea..0461ba3e82db 100644
--- a/arch/arm64/boot/dts/rockchip/rk3399-rv1-android.dts
+++ b/arch/arm64/boot/dts/rockchip/rk3399-rv1-android.dts
@@ -44,7 +44,7 @@
 #include <dt-bindings/pwm/pwm.h>
 #include <dt-bindings/sensor-dev.h>
 #include "rk3399.dtsi"
-#include "rk3399-android.dtsi"
+#include "rk3399-android-6.0.dtsi"
 #include "rk3399-opp.dtsi"
 
 / {
diff --git a/arch/arm64/boot/dts/rockchip/rk3399-sapphire-excavator-box.dts b/arch/arm64/boot/dts/rockchip/rk3399-sapphire-excavator-box-android-6.0.dts
similarity index 99%
rename from arch/arm64/boot/dts/rockchip/rk3399-sapphire-excavator-box.dts
rename to arch/arm64/boot/dts/rockchip/rk3399-sapphire-excavator-box-android-6.0.dts
index ec08bf870e99..d2d79b1eb16d 100644
--- a/arch/arm64/boot/dts/rockchip/rk3399-sapphire-excavator-box.dts
+++ b/arch/arm64/boot/dts/rockchip/rk3399-sapphire-excavator-box-android-6.0.dts
@@ -44,7 +44,7 @@
 
 #include <dt-bindings/input/input.h>
 #include "rk3399-excavator-sapphire.dtsi"
-#include "rk3399-android.dtsi"
+#include "rk3399-android-6.0.dtsi"
 
 / {
 	compatible = "rockchip,rk3399-excavator-box", "rockchip,rk3399";
diff --git a/arch/arm64/boot/dts/rockchip/rk3399-sapphire-excavator-disvr.dts b/arch/arm64/boot/dts/rockchip/rk3399-sapphire-excavator-disvr.dts
index ba077312a269..c3ba9a067e2a 100644
--- a/arch/arm64/boot/dts/rockchip/rk3399-sapphire-excavator-disvr.dts
+++ b/arch/arm64/boot/dts/rockchip/rk3399-sapphire-excavator-disvr.dts
@@ -44,7 +44,7 @@
 
 #include <dt-bindings/input/input.h>
 #include "rk3399-excavator-sapphire.dtsi"
-#include "rk3399-android.dtsi"
+#include "rk3399-android-6.0.dtsi"
 
 / {
 	compatible = "rockchip,rk3399-excavator-edp", "rockchip,rk3399";
diff --git a/arch/arm64/boot/dts/rockchip/rk3399-sapphire-excavator-edp.dts b/arch/arm64/boot/dts/rockchip/rk3399-sapphire-excavator-edp-android-6.0.dts
similarity index 99%
rename from arch/arm64/boot/dts/rockchip/rk3399-sapphire-excavator-edp.dts
rename to arch/arm64/boot/dts/rockchip/rk3399-sapphire-excavator-edp-android-6.0.dts
index 8aadb7f65b9a..0d0375b40bb5 100644
--- a/arch/arm64/boot/dts/rockchip/rk3399-sapphire-excavator-edp.dts
+++ b/arch/arm64/boot/dts/rockchip/rk3399-sapphire-excavator-edp-android-6.0.dts
@@ -44,7 +44,7 @@
 
 #include <dt-bindings/input/input.h>
 #include "rk3399-excavator-sapphire.dtsi"
-#include "rk3399-android.dtsi"
+#include "rk3399-android-6.0.dtsi"
 
 / {
 	compatible = "rockchip,rk3399-excavator-edp", "rockchip,rk3399";
diff --git a/arch/arm64/boot/dts/rockchip/rk3399-vr-android.dts b/arch/arm64/boot/dts/rockchip/rk3399-vr-android.dts
index 4472baa0115a..f1b7e64f3ce7 100644
--- a/arch/arm64/boot/dts/rockchip/rk3399-vr-android.dts
+++ b/arch/arm64/boot/dts/rockchip/rk3399-vr-android.dts
@@ -44,7 +44,7 @@
 #include <dt-bindings/pwm/pwm.h>
 #include <dt-bindings/sensor-dev.h>
 #include "rk3399.dtsi"
-#include "rk3399-android.dtsi"
+#include "rk3399-android-6.0.dtsi"
 #include "rk3399-opp.dtsi"
 
 / {
-- 
2.35.3

